<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimalsForm</class>
 <widget class="QWidget" name="AnimalsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>371</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Птахи</string>
   </property>
   <widget class="QTableView" name="birdsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>351</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>371</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Савці</string>
   </property>
   <widget class="QTableView" name="mammalsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>351</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>1131</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Вибрати</string>
   </property>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>90</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Видалити</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Знайти</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="idText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>60</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Редагувати</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>79</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>                 Унікальний ідентифікатор</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="searchAnimalGroupBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>371</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Савець/Птаха/Рептилія</string>
    </property>
    <widget class="QTableView" name="Viewer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>351</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="mammalCheckBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Сcавець</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="birdCheckBox_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Птаха</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="reptileCheckBox_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Рептилія</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>1131</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Внести тварину</string>
   </property>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Тип тварини</string>
    </property>
    <widget class="QCheckBox" name="mammalCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>78</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Сcавець</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="reptileCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>78</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Рептилія</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="birdCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>78</width>
       <height>22</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Птаха</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>981</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Заповнення</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <widget class="QGroupBox" name="sampleGroupBox">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>261</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>Вибрати</string>
     </property>
     <widget class="QComboBox" name="rationComboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="rationTypeComboBox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="caretakerComboBox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="veterinarianComboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="livANComboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>         Доглядач</string>
      </property>
      <property name="textFormat">
       <enum>Qt::MarkdownText</enum>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>        Ветеринар</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>80</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>            Раціон</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>        Тип раціону</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> Зона проживання</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="jointGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>Спільні поля</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Унікальний ідентифікатор</string>
      </property>
      <property name="margin">
       <number>3</number>
      </property>
     </widget>
     <widget class="QTextEdit" name="identifierText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ім'я</string>
      </property>
      <property name="margin">
       <number>61</number>
      </property>
     </widget>
     <widget class="QTextEdit" name="nameText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="ageText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Вік</string>
      </property>
      <property name="margin">
       <number>63</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Стать</string>
      </property>
      <property name="margin">
       <number>59</number>
      </property>
     </widget>
     <widget class="QTextEdit" name="sexText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>230</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Внести дані</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="birdInfoGroupBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>171</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>Відомості про місце зимівлі</string>
     </property>
     <widget class="QTextEdit" name="countryCodeText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Код країни</string>
      </property>
      <property name="margin">
       <number>45</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Назва країни</string>
      </property>
      <property name="margin">
       <number>40</number>
      </property>
     </widget>
     <widget class="QTextEdit" name="countryNameText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Дата початку</string>
      </property>
      <property name="margin">
       <number>40</number>
      </property>
     </widget>
     <widget class="QTextEdit" name="startDateBText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Дата завершення</string>
      </property>
      <property name="margin">
       <number>30</number>
      </property>
     </widget>
     <widget class="QTextEdit" name="endDateBText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="reptileInfoGroupBox">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>171</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>Відомості </string>
     </property>
     <widget class="QTextEdit" name="avarageTemperatureText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>    Середня температура</string>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>   Дата поч. зим. сплячки</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="stardDateRText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="endDateRText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>   Дата зав. зим. сплячки</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_11">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>40</y>
     <width>371</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Рептилії</string>
   </property>
   <widget class="QTableView" name="reptilesView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>351</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
